AutoDarkModeApp/Properties/Resources.pl.resx

```diff
@@ -379,10 +379,10 @@ Podaruj tyle, na ile cenisz moją pracę.</value>
     <value>Alternatywny biały motyw</value>
   </data>
   <data name="cbSettingsColourFilter" xml:space="preserve">
-    <value>Użyj szary filtr kolorów (wciska Win + Ctrl + C)</value>
+    <value>Użyj szarego filtru kolorów (Win + Ctrl + C)</value>
   </data>
   <data name="cbSettingsLogonTask" xml:space="preserve">
-    <value>Użyj harmonogramu zamiast autouruchamiania</value>
+    <value>Użyj harmonogramu Windowsa zamiast autouruchamiania</value>
```

Comment:
More specific, since harmonogram means windows scheduler and is also word for schedule in the app